File: 8KYU/White or Black?/solution.py
def square_color(file, rank):
    if file == "a" or file == "c" or file == "e" or file == "g":
        if(rank % 2 == 0):
            return "white"
        else:
            return "black"
    else:
        if(rank % 2 == 0):
            return "black"
        else:
            return "white"

File: 8KYU/White or Black?/solution.js
function mineColor(file, rank) {
    if(file == "a" || file == "c" || file == "e" || file == "g"){
      return (rank % 2 == 0) ? 'white' : 'black';
    } else {
      return (rank % 2 == 0) ? 'black' : 'white';
    }  
}